google/cloud/bigquery_storage_v1/reader.py

```diff
@@ -116,11 +125,12 @@ def __init__(self, wrapped, client, name, offset, read_rows_kwargs):
 
         # Make a copy of the read position so that we can update it without
         # mutating the original input.
-        self._wrapped = wrapped
         self._client = client
         self._name = name
         self._offset = offset
         self._read_rows_kwargs = read_rows_kwargs
+        self._retry_delay_callback = retry_delay_callback
+        self._reconnect()
```

Comment:
Doing heavy work during construction/initialization time is a bit of an OO anti-pattern. I'd prefer we find another way to do this.

Reply:
Removed this from construction and added it as an additional statement to wherever ReadRowsStream is constructed previously.